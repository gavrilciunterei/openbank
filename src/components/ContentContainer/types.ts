export type ContentContainerProps = {
  children: React.ReactNode;
};
