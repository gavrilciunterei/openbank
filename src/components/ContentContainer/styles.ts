import styled from 'styled-components';

export const ContainerPadding = styled.div`
  padding: 20px 40px 20px 40px;

  @media (min-width: 800px) {
    padding: 20px 100px 20px 100px;
  }
`;
