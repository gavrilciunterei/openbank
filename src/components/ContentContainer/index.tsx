import React from 'react';
import { ContainerPadding } from './styles';
import { ContentContainerProps } from './types';

function ContentContainer({ children }: ContentContainerProps) {
  return <ContainerPadding>{children}</ContainerPadding>;
}

export default ContentContainer;
