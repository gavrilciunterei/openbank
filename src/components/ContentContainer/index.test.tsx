import React from 'react';
import renderer from 'react-test-renderer';
import ContentContainer from './index';

it('Test ContentContainer', () => {
  const tree = renderer
    .create(
      <ContentContainer>
        <div>
          <h1> Hola </h1>
        </div>
      </ContentContainer>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
