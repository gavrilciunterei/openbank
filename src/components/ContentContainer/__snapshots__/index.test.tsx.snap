// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ContentContainer 1`] = `
<div
  className="sc-bdvvtL gMSJIZ"
>
  <div>
    <h1>
       Hola 
    </h1>
  </div>
</div>
`;
